model/project.go

```
@@ -576,14 +576,26 @@ var (
 	ProjectTasksKey         = bsonutil.MustHaveTag(Project{}, "Tasks")
 )
 
-func PopulateExpansions(t *task.Task, h *host.Host) (util.Expansions, error) {
+func PopulateExpansions(t *task.Task, h *host.Host, settings *evergreen.Settings) (util.Expansions, error) {
 	if t == nil {
 		return nil, errors.New("task cannot be nil")
 	}
 	if h == nil {
 		return nil, errors.New("host cannot be nil")
 	}
+
 	expansions := util.Expansions{}
+
+	if _, ok := settings.Credentials["github"]; ok {
```

I would just call the function and ignore the error it returns